test/redis/interval_cache/getter_test.exs

```diff
@@ -0,0 +1,105 @@
+defmodule Castle.RedisIntervalCacheGetterTest do
+  use Castle.RedisCase, async: false
+  use Castle.TimeHelpers
+
+  @moduletag :redis
+
+  import Mock
+  alias Castle.Redis.Conn, as: Conn
+  alias Castle.Redis.Interval.Keys, as: Keys
+  alias Castle.Redis.Interval.Getter, as: Getter
+
+  @prefix "interval.cache.getter.test"
+
+  setup do
+    redis_clear("#{@prefix}*")
+    from = get_dtim("2017-03-22T01:15:00Z")
+    to = get_dtim("2017-03-22T02:15:00Z")
+    rollup = BigQuery.TimestampRollups.QuarterHourly
+    keys = Keys.keys(@prefix, rollup.range(from, to, false))
+    [from: from, to: to, rollup: rollup, keys: keys]
+  end
+
+  test "hits the entire range", %{from: from, to: to, rollup: rollup, keys: keys} do
+    Conn.hset(Enum.at(keys, 0), "key1", 11)
```

Comment:
might vote for one of these `key`s being renamed? maybe `keys` could be named `cached_intervals` or something? or `key1` etc could be `podcast1`? a little confusing at first glance otherwise

Reply:
+yes